src/Compute/Compute.Sdk/README.md

```diff
@@ -0,0 +1,82 @@
+# Overall
+This directory contains the service clients of Az.Compute module.
+
+## Run Generation
+In this directory, run AutoRest:
+```
+autorest.cmd README.md --version=v2
+```
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+csharp: true
+clear-output-folder: true
+reflect-api-versions: true
+openapi-type: arm
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+
+title: ComputeManagementClient
+payload-flattening-threshold: 1
+
+directive:
+    # dynamically add a DummyOrchestrationServiceName value to the enum 
+  - from: virtualMachineScaleSet.json
+    where: $..enum
+    transform: >-
+      if( $.length === 1 && $[0] === "AutomaticRepairs") { 
+        $.push('DummyOrchestrationServiceName');
+      }
+      return $;
+    
+    # remove it from the C# generated code
+  - from: source-file-csharp
+    where: $ 
+    transform: >-
+      return $.
+        replace(/.*public const string DummyOrchestrationServiceName.*/g,'').
+        replace(/, 'DummyOrchestrationServiceName'/g,'');
+  - from: gallery.json
+    where: $.definitions.GalleryTargetExtendedLocation.properties.storageAccountType["x-ms-enum"].name
+    transform: return "EdgeZoneStorageAccountType"
+```
+
+
+### 
+``` yaml
+commit: 6107f258542bc16f1fdb5c377225246b71ce2143
+input-file: 
+  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/common-types/v1/common.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/computeRPCommon.json
```

Comment:
Please replace `main` with $(commit) for all the URLs.

Reply:
somehow this part did not get commited.  Added now